test/Analysis/test-allocation.mlir

```diff
@@ -875,4 +889,26 @@ tt.func @two_different_ws() {
   tt.return
 }
 
+// expected-remark @below {{ptr_allocation_datalayout}}
+// expected-remark @below {{size = 8}}
+tt.func @ptr_allocation_datalayout(%arg0: !tt.ptr<i32>) {
```

Comment:
Do you still need this test?

Reply:
Yes, but now it's just testing that PointerType can have its size queried (just not through `llvm::DataLayout`)